esque/controller/topic_controller.py

```diff
@@ -120,8 +121,24 @@ def update_from_cluster(self, topic: Topic):
 
         topic.is_only_local = False
 
+        message = self._get_last_message(topic)
+        topic.last_message_timestamp = message.timestamp()[1]
+
         return topic
 
+    def _get_last_message(self, local_topic: Topic) -> Optional[Tuple[str, int]]:
```

Comment:
The message with the highest offset isn't necessarily also the latest message. You'll have to go through all partitions and look at the last message to find out which one is the latest in time.

Reply:
Ah okay that's good to know. It makes the code even "smaller" because I won't need the `max_offset` method in `topic.py`